lib/turbo/topup/views/topup_modal.dart

```diff
@@ -82,6 +85,8 @@ void showTurboModal(BuildContext context, {Function()? onSuccess}) {
       onSuccess?.call();
     }
 
+    context.read<ActivityTracker>().setToppingUp(false);
```

Comment:
Will change if you replace the method approach to the setter approach
```suggestion
    context.read<ActivityTracker>().setToppingUp = false;
```